report/3.Theory/1.TrackingObjects.tex

```diff
@@ -0,0 +1,25 @@
+\section{Predicting a moving objects location}
+In order to predict a moving objects location a keen observation to be made is whether the object is flying i.e. if the object is following a ballistic trajectory or whether the object is driving i.e. following a linear trajectory. 
+In the following section we will elaborate on these distinctions in order to gain a deeper understanding on the subject and to be able to complete our objectivs. 
+\subsection{Linear trajectory}
+Given that the target follows a linear trajectory a simple method to calculate the new location of the target can be applied. 
+By using the image processing method as descibed in Section~\ref{} locating the target is considered irrelevent for this subsection. 
+In order to predict our targets location we first need to understand some basic physics of kinematics. 
```

Comment:
We generally try to avoid the use of "our", "we", "you", "us" etc. But I guess that should be a discussion for the group, how and when we will allow those.